src/routes/opening/index.tsx

```diff
@@ -137,6 +137,8 @@ export const SafeDeployment = ({
     (err: Error) => {
       if (isTxPendingError(err)) {
         dispatch(enqueueSnackbar({ ...NOTIFICATIONS.TX_PENDING_MSG }))
+      } else {
+        dispatch(enqueueSnackbar({ ...NOTIFICATIONS.CREATE_SAFE_FAILED_MSG }))
```

Comment:
I think it wasn't being done before because the interface itself shows an error.

Reply:
I didn't see the notification

Reply:
There was no notification, I meant that the page itself shows an error graphic and suggests to retry.
Notification doesn't hurt though.